libs/api/domains/national-registry/src/lib/nationalRegistry.service.ts

```diff
@@ -28,6 +29,9 @@ export class NationalRegistryService {
       fullName: user.Fulltnafn,
       gender: this.formatGender(user.Kyn),
       maritalStatus: this.formatMaritalStatus(user.hju),
+      spouseName: user.nafnmaka,
+      spouseNationalId: user.MakiKt,
+      spouseCohab: user.Sambudarmaki,
```

Comment:
Namespace in string, should be an object, see comment on type

Reply:
Yes, makes sense!